packages/now-cli/src/util/dev/server.ts

```diff
@@ -1706,7 +1710,29 @@ export default class DevServer {
 
     this.output.debug(`Spawning dev command: ${command}`);
 
-    const p = spawnCommand(command, { stdio: 'inherit', cwd, env });
+    const p = spawnCommand(command, {
+      stdio: [process.stdin, 'pipe', 'pipe'],
+      cwd,
+      env,
+    });
+
+    if (!p.stdout || !p.stderr) {
+      throw new Error('Expected child process to have stdout and stderr');
+    }
+
+    p.stdout.on('data', data => {
+      if (data) {
+        this.output.debug(`Replacing framework output with now dev address`);
+        data = data
+          .toString()
```

Comment:
We should invoke `p.stdout.setEncoding('utf8')` rather than `.toString()` on every Buffer.

Reply:
Fixed